web/src/components/NavigationBarScrollContainer.tsx

```diff
@@ -59,8 +60,26 @@ const getScrollableWidth = (scrollContainer: RefObject<HTMLDivElement>): number
   return 0
 }
 
-const NavigationBarScrollContainer = ({ children, direction, scrollContainerRef }: PropsType): ReactElement => {
-  const [scrollPosition, setScrollPosition] = useState(0)
+const NavigationBarScrollContainer = ({
+  children,
+  direction,
+  scrollContainerRef,
+  activeIndex,
+}: PropsType): ReactElement => {
+  const getInitScrollPosition = useCallback((): number => {
+    const navBar = document.getElementById('navbar')
+    if (!navBar) {
+      return 0
+    }
+    const navBarOffset = navBar.offsetLeft
+    const elementOffset = navBar.getElementsByTagName('div')[activeIndex]?.offsetLeft
+    return elementOffset ? navBarOffset + elementOffset : 0
+  }, [activeIndex])
+  const [scrollPosition, setScrollPosition] = useState<number>(activeIndex > 0 ? getInitScrollPosition() : 0)
```

Comment:
This is never updated, is it? So there should be no need to save it in a state.

Reply:
> When testing in Firefox with a screen width of 372px, I can only see the very beginning of the selected icon (which is what I see at scroll 0).
> 
> <img alt="image" width="434" src="https://user-images.githubusercontent.com/18513943/193845469-4c2df55d-ce1f-4d70-8fd1-9c31b1b0ac28.png">

okay looks like the solution is only working on chrome...have to check

Reply:
@steffenkleinle i adjusted some things and added a custom hook to deal with ref changes. Should work now also on forefox